README.md

```diff
@@ -56,6 +60,18 @@ Tool                  | Description
 [Kaminari]            | Elegant pagination for any sort of collection
 [Ransack]             | Provides a simple search API to query your data
 
+## Acknowledgements
+
+Thanks to [Greg] for creating and opensourcing this project.
+
+Thanks to all the people that ever contributed to this project through [code] or
```

Comment:
I think we can improve this by linking the following instead:

```suggestion
Thanks to [all the people that ever contributed to this project through code][contributors] or
```

I'm not familiar with this link markdown syntax so I may have it wrong, sorry. I also renamed the link to contributors as I felt that was more reflective of what it is.